src/torcontrol.cpp

```diff
@@ -134,12 +134,17 @@ bool TorControlConnection::Connect(const std::string& tor_control_center, const
 {
     if (b_conn)
         Disconnect();
```

Comment:
Don't hesitate to fix this if you retouch (per doc/developer-notes.md about code style for conditionals)
```diff
-    if (b_conn)
+    if (b_conn) {
         Disconnect();
+    }
```